GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js

```diff
@@ -261,15 +261,19 @@ gdjs.RuntimeGamePixiRenderer.prototype.setFullScreen = function (enable) {
       // Use HTML5 Fullscreen API
       //TODO: Do this on a user gesture, otherwise most browsers won't activate fullscreen
       if (this._isFullscreen) {
-        if (document.documentElement.requestFullScreen) {
+        if (document.documentElement.requestFullscreen) {
+          document.documentElement.requestFullscreen();
+        } else if (document.documentElement.requestFullScreen) {
```

Comment:
I suppose it's possible that `requestFullScreen` is a typo, and that it should be `requestFullscreen` instead.

ie. that we don't need to retain `requestFullScreen` as an option.

Reply:
Possibly! Must have been a mistake because the old `webkit` and `moz` prefixed versions are `xxxRequestFullScreen`. Let's remove `requestFullScreen` then :)

Reply:
@jonasfj Do you want to remove the references to requestFullScreen?

Reply:
Done